### PR TITLE
Add graph controls and fix axes

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* Add tickmarks to histogram sliders
* Fix the vertical axis height so that the whole graph doesn't redraw as you use slider
   * TODO - make this work after changing smoothing

image

**Release Note**

```release-note
Key server histograms have better controls and don't resize as days change
```

### CODE REVIEW

File: cmd/server/assets/realmadmin/_stats_keyserver.html [hunk 1]
```diff
@@ -100,10 +100,26 @@ <h5 class="modal-title">Key server stats</h5>
...
       $slider.val(data.length);
       $onsetSlider.attr('min', Math.min(smoothing, data.length));
       $onsetSlider.attr('max', data.length);
+      $onsetSlider.attr('value',data.length);
+      $onsetSlider.attr('step', 1);
       $onsetSlider.val(data.length);
+  
+      $("#tektickmarks").empty();
+      for (i = Math.min(smoothing, data.length); i < data.length; i++) {
+        let opt = '<option value="' + i + '" label="' + data[i].day.substring(0,10) + '"></option>';
```
Comment (whaught) [inline cmd/server/assets/realmadmin/_stats_keyserver.html:114]: I think its preferred to make an option and mess with it's features one-by-one
```
let $opt = $('<option/>');
$opt.val(i);
$opt.attr('label', substring);

foo.append($opt);
```

File: cmd/server/assets/realmadmin/_stats_keyserver.html [hunk 2]
```diff
@@ -100,10 +100,26 @@ <h5 class="modal-title">Key server stats</h5>
...
+        $("#tektickmarks").append(opt);
+      }
+
+      $("#onset_slide_ticks").empty();
```
Comment (whaught) [inline cmd/server/assets/realmadmin/_stats_keyserver.html:118]: every time you do `$('#onset_slide_ticks')` jquery is retrieving it from the DOM. I'd just make this once and reference a variable here.

File: cmd/server/assets/realmadmin/_stats_keyserver.html [hunk 3]
```diff
@@ -308,18 +338,26 @@ <h5 class="modal-title">Key server stats</h5>
...
 
     if (!data.length) {
       $div.find('p').text('No data yet.');
       return;
     }
     stats = data;
 
+    // calculate the vaxis max.
+    for (i = 0; i < stats.length - smoothing; i++) {
+      getOnsetDataTable(i);
```
Comment (whaught) [inline cmd/server/assets/realmadmin/_stats_keyserver.html:358]: ah now 'getOnsetDataTable' creates a bunch of new google.visualization.DataTable() to be immediately thrown away (to populate the global variable). This is getting messy.

If we want to do this maybe we should cache the results so we don't need to rebuild these as we slide the slider.

**Review by whaught — APPROVED** [review]

File: cmd/server/assets/realmadmin/_stats_keyserver.html (hunk 3 above)
Comment (mikehelmick) [reply cmd/server/assets/realmadmin/_stats_keyserver.html:358]: added caches - redraws are faster now!

**Review by whaught — APPROVED** [review]

**Pull request merged**